CONTRIBUTING.rst

```diff
@@ -201,32 +207,68 @@ business days.
 You can find all `open pull requests`_ on GitHub and open `Zephyr Project
 Issues`_ in Github issues.
 
-Development Tools and Git Setup
-*******************************
+ .. _Continuous Integration:
+
+Continuous Integration (CI)
+***************************
+
+The Zephyr Project operates a Continuous Integration (CI) system that runs on
+every Pull Request (PR) in order to verify several aspects of the PR:
+
+* Git commit formatting
+* Coding Style
+* Sanity Check builds for multiple architectures and boards
+
+CI is run on the ``shippable`` cloud service and it uses the same tools
+described in the `Contribution Tools`_ section. The CI result must be green in
```

Comment:
How about:

    The CI results must be green indicating "All checks have passed" before
    the Pull Request can be merged.  CI is run when the PR is created, and 
    again every time the PR is modified with a commit.  You can also force
    the CI system to recheck a PR by adding a comment to the PR saying
    simply ``retest`` in the message (helpful if the CI system fails unexpectedly).

Reply:
Fixed